apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -339,22 +351,86 @@
                                     </div>
                                     <br>
                                 <% } else { %>
-                                    <div class="external-login blurring external-login-dimmer">
-                                        <div class="field">
-                                            <button
-                                                class="ui button fluid"
-                                                onclick="handleNoDomain(this,
-                                                    '<%=Encode.forJavaScriptAttribute(Encode.forUriComponent(idpName))%>',
-                                                    '<%=Encode.forJavaScriptAttribute(Encode.forUriComponent(idpEntry.getValue()))%>')"
-                                                id="icon-<%=iconId%>"
-                                                title="<%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with")%> <%=Encode.forHtmlAttribute(idpName)%>"
-                                            >
-                                                <img class="ui image" src="<%=Encode.forHtmlAttribute(imageURL)%>">
-                                                <span><%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with")%> <%=Encode.forHtmlContent(idpName)%></span>
-                                            </button>
+                                    <% if (StringUtils.equals(idpEntry.getValue(),GOOGLE_AUTHENTICATOR)) {
+                                            isGoogleIdp = true;
+                                            IdentityProviderDataRetrievalClient identityProviderDataRetrievalClient = new IdentityProviderDataRetrievalClient();
+                                            List<String> configKeys = new ArrayList<>();
+                                            configKeys.add("ClientId");
+                                            configKeys.add("callbackUrl");
+                                            configKeys.add("IsGoogleOneTapEnabled");
```

Comment:
Better to use a same naming convention for all the properties. ie. `callbackUrl` is in camel case.

Reply:
This is driven by the authenticator properties. So we cannot change it in this level. WDYT ?
{
  "name": "GoogleOIDCAuthenticator",
  "isEnabled": true,
  "isDefault": true,
  "properties": [
    {
      "key": "callbackUrl",
      "value": "https://dev.api.asgardeo.io/commonauth"
    },
    {
      "key": "AdditionalQueryParameters",
      "value": "scope=email openid profile&login_hint=${login_hint}&prompt=select_account"
    },
    {
      "key": "ClientId",
      "value": "876*"
    },
    {
      "key": "ClientSecret",
      "value": "*"
    },
    {
      "key": "IsGoogleOneTapEnabled",
      "value": "true"
    }
  ]
}